mods/cnc/weapons/missiles.yaml

```diff
@@ -178,15 +122,11 @@ MammothMissiles:
 	Report: rocket1.aud
 	ValidTargets: Ground
 	Projectile: Bullet
-		Arm: 5
-		Blockable: false
 		Shadow: true
 		Inaccuracy: 853
 		LaunchAngle: 62
 		Image: DRAGON
-		RateOfTurn: 2
```

Comment:
`RateOfTurn` was bit-rot, but I guess we probably want the internal default of 5 here rather than 15 to match bleed.